assets/components/numberInput/numberInput.scss

```diff
@@ -11,30 +11,30 @@
 	&::placeholder {
 		color: gu-colour(neutral-1);
 	}
+}
 
-	.component-number-input__label {
-		position: absolute;
-		top: 7px;
-		left: $gu-h-spacing;
-	}
+.component-number-input__label {
```

Comment:
Since you are flattening this file here. Is it worth to change tabs for two spaces indentation to try to tend to a standard?

Reply:
Shall we agree that going forward our standard for CSS is two spaces then?

Reply:
I am more than happy to go with two spaces 👍

Reply:
I created a PR to add two-spaces into the CSS-guidelines: https://github.com/guardian/support-frontend/pull/486

Reply:
Updated.